src/Components/ContainerItem/ContainerItem.jsx

```diff
@@ -86,20 +87,35 @@ export default function ContainerItem({
 		}
 	}, [walletData]); // eslint-disable-line react-hooks/exhaustive-deps
 
-	const onGetObjects = (containerId, pageTemp = pagination.page) => {
-		setPagination({ ...pagination, page: pageTemp});
+	const onGetObjects = (containerId, cursor = '', paginationTemp = pagination, isTreeViewObjectsTemp = isTreeViewObjects) => {
 		setLoadingObjects(true);
-		api('POST', `v1/objects/${containerId}/search?limit=${ObjectsPerPage}&offset=${pageTemp * ObjectsPerPage}`, {
-			filters: filters.every((item) => item.key !== '' && item.value !== '') ? filters : [],
+		const filtersApplied = filters.filter(item => item.key !== '' && item.value !== '');
+		const hasFileName = filtersApplied.some(item => item.key === 'FileName');
+		let filtersFinal = hasFileName ? filtersApplied : [{ key: "FileName", match: "MatchCommonPrefix", value: "" }, ...filtersApplied];
```

Comment:
Why are we using `FileName` filter here by default?

Reply:
If we don't specify, we won't receive attributes with file names (`attributes: {}`) for ui, is it ok?

<img width="394" alt="image" src="https://github.com/user-attachments/assets/7d33cc86-c73a-4f52-bc6b-98d304598402" />

Reply:
@roman-khimov done